### DISCUSSION TITLE
The pnpm-win-x64.exe installation package cannot be installed on window11

### DISCUSSION THREAD

**Discussion by sunshine-we** [pnpm/pnpm#8310] [Q&A]
window version: win11
node version: 20.15.1  & 18.20.4
pnpm version: 9.5.0 
download link：[pnpm-win-x64.exe](https://github.com/pnpm/pnpm/releases/download/v9.5.0/pnpm-win-x64.exe)
When I download [pnpm-win-x64.exe\ to my local computer and double-click it to open it, I cannot jump to the installation interface.

**Comment by zkochan** [comment]
It isn't an installer. It is the pnpm CLI itself.

To install it, run:

```
iwr https://get.pnpm.io/install.ps1 -useb | iex
```
